addons/l10n_in_edi_ewaybill/views/account_move_views.xml

```diff
@@ -16,14 +16,7 @@
                     <group name="ewaybill_group">
                         <group string="Transaction Details" name="Transaction_group" 
                             attrs="{'invisible': [('l10n_in_edi_ewaybill_direct_api', '!=', True)]}">
-                            <field name="l10n_in_type_id"
-                                attrs="{'invisible': [
-                                    ('move_type','in',('in_invoice','in_refund','in_receipt'))]}"
-                                domain="[('allowed_supply_type','in',('in','both'))]"/>
-                            <field name="l10n_in_type_id"
-                                attrs="{'invisible': [
-                                    ('move_type','in',('out_invoice','out_refund','out_receipt'))]}"
-                                domain="[('allowed_supply_type','in',('out','both'))]"/>
+                            <field name="l10n_in_type_id" context="{'l10n_in_type_search_mode': (context.get('default_move_type', 'entry') in ('out_invoice', 'out_refund', 'out_receipt') and 'out') or (context.get('default_move_type', 'entry') in ('in_invoice', 'in_refund', 'in_receipt') and 'in') or False}"/>
```

Comment:
will this work without a module update?

Reply:
no, and it's just merged so no worry about that